src/styles/_variables.scss

```diff
@@ -0,0 +1 @@
+$gap: 48px;
```

Comment:
no need for a variable that you only use once
```suggestion
```

Reply:
But it can be usefull in the future